proposals/pubsub.md

```diff
@@ -0,0 +1,182 @@
+# PubSub enabled by default
+
+Authors:
+- @gozala
+
+Initial PR: https://github.com/protocol/web3-dev-team/pull/53
+
+<!--
+This template is for a proposal/brief/pitch for a significant project to be undertaken by a Web3 Dev project team.
+The goal of project proposals is to help us decide which work to take on, which things are more valuable than other things.
+-->
+<!--
+A proposal should contain enough detail for others to understand how this project contributes to our team’s mission of product-market fit
+for our unified stack of protocols, what is included in scope of the project, where to get started if a project team were to take this on,
+and any other information relevant for prioritizing this project against others.
+It does not need to describe the work in much detail. Most technical design and planning would take place after a proposal is adopted.
+Good project scope aims for ~3-5 engineers for 1-3 months (though feel free to suggest larger-scoped projects anyway). 
+Projects do not include regular day-to-day maintenance and improvement work, e.g. on testing, tooling, validation, code clarity, refactors for future capability, etc.
+-->
+<!--
+For ease of discussion in PRs, consider breaking lines after every sentence or long phrase.
+-->
+
+
+
+## Purpose &amp; impact 
+#### Background &amp; intent
+_Describe the desired state of the world after this project? Why does that matter?_
+<!--
+Outline the status quo, including any relevant context on the problem you’re seeing that this project should solve.
+Wherever possible, include pains or problems that you’ve seen users experience to help motivate why solving this problem works towards top-line objectives.
+-->
+
+As of this writing [pubsub][IPFS pubsub] is considered experimental and is disabled in go-ipfs.
+
+> Can be enabled by starting a daemon with `--enable-pubsub-experiment` flag
+
+In practice however many teams building products on IPFS use and depend on pubsub:
+
+- Textile nodes have pubsub enabled.
+- Ceramic uses pubsub for queries and log replication.
+- [OrbitDB][] depends on pubsub to do replication.
+- Fission team would like to use pubsub (has user requests, but found it unreliable)
+
+It also fragments network by (JS) nodes having it enabled and nodes that do not making IPNS unrelaible.
+
+> IPFS in browsers use IPNS over pubsub (no DHT for browsers) while other nodes do it over DHT.
+
+Enabling PubSub would populate network with more nodes making it more reliable.
```

Comment:
is this accurate given Steven's statement above? aka there will be no impact on existing pubsub topics' reliability unless additional nodes subscribe - correct?

Reply:
Correct, this is not accurate. There may be a side effect of more people using pubsub and subscribing to particular topics and therefore increasing the mesh size and resiliency of those topics.

Reply:
```suggestion
Enabling PubSub would populate network with more nodes, which might make specific popular topics more resilient / reliable.
```